controllers/homework.js

```diff
@@ -128,15 +128,28 @@ const getCreateHandler = (service) => (req, res, next) => {
 				message: res.$t('homework._task.text.startDateBeforeSubmissionDate'),
 			};
 			redirectHelper.safeBackRedirect(req, res);
-			return;
+			return false;
 		}
 	}
 
-	handleTeamSubmissionsBody(req.body, res.locals.currentUser);
+	return true;
+};
+
+const prepareRequestBodyForSubmission = (service, req, res) => {
+	if (service === 'submissions') {
+		handleTeamSubmissionsBody(req.body, res.locals.currentUser);
 
-	if (req.body.teamMembers && typeof req.body.teamMembers === 'string') {
-		req.body.teamMembers = [req.body.teamMembers];
+		if (req.body.teamMembers && typeof req.body.teamMembers === 'string') {
+			req.body.teamMembers = [req.body.teamMembers];
+		}
 	}
+};
+
+const getCreateHandler = (service) => (req, res, next) => {
+	if (!prepareRequestBodyForHomework(service, req, res)) return;
```

Comment:
Ist legacy code, ich weiß, aber wollt ihr nicht diesen Date-Check in ne andere Methode verschieben, wenn ihr das eh gerade ändert? Dass es dann ungefähr so aussieht:
```suggestion
	prepareRequestBodyForSubmission(service, req, res);
	
	if(!isStartDateBeforeDueDate(req)) { return } 
```
Das liest sich sehr seltsam, dass die prepare-Methode ein boolean zurück gibt.